src/main/java/uk/gov/dwp/dataworks/controller/DataKeyController.java

```diff
@@ -4,21 +4,24 @@
 import io.swagger.annotations.ApiOperation;
 import io.swagger.annotations.ApiResponse;
 import io.swagger.annotations.ApiResponses;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.http.HttpStatus;
 import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
 import uk.gov.dwp.dataworks.dto.DecryptDataKeyResponse;
 import uk.gov.dwp.dataworks.dto.GenerateDataKeyResponse;
 import uk.gov.dwp.dataworks.service.DataKeyService;
-import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.web.bind.annotation.*;
 
 @SuppressWarnings("unused")
 @RestController
 @RequestMapping("/datakey")
 @Api(value="datakey")
 public class DataKeyController {
 
-    private DataKeyService dataKeyService;
+    private final DataKeyService dataKeyService;
+    private final static Logger LOGGER = LoggerFactory.getLogger(DataKeyController.class);
```

Comment:
This logger does not seem to be in use

Reply:
Removed unused Logger from DatakeyController